build.gradle

```diff
@@ -67,6 +67,8 @@ subprojects {
         compile 'org.reactivestreams:reactive-streams:1.0.0'
         compile 'org.slf4j:slf4j-api:1.7.25'
         compile 'com.google.code.findbugs:jsr305:3.0.0'
+        compile 'com.fasterxml.jackson.core:jackson-core:2.8.8'
```

Comment:
not thrilled about the idea of making jackson a dependency of rsocket

Reply:
Yep - I will pull that out and not provide any implementation apart from NONE in core.